cluster-controller/src/main/java/io/strimzi/controller/cluster/operations/cluster/AbstractClusterOperations.java

```diff
@@ -230,6 +232,7 @@ public final void reconcile(String namespace, String name) {
         final String lockName = getLockName(clusterType, namespace, name);
         vertx.sharedData().getLockWithTimeout(lockName, LOCK_TIMEOUT, res -> {
             if (res.succeeded()) {
+                log.debug("Lock acquired");
```

Comment:
Better to include the lock name in the message, for this and releases

Reply:
Agree